### ISSUE TITLE
[feature]: In Initiative section add image related to lag line.

### ISSUE THREAD

**Issue by starrohan999** [garg3133/JagratiWebApp#187]
**Is your feature request related to a problem? Please describe.**
  In the initiative section, all the images are the same for all cards and they doest not match their tagline. I think that makes an impact on the viewer who watches this.

**Describe the solution you'd like**
Simply add an image related to the tagline.
Ex: the tagline is BOOK DISTRIBUTION so we add an image that shows us that we had done this thing before.
     there are many tag lines but the images are the same so.

**Additional context**
looking at the plantation drive tagline it does not match with background image
![2021-03-22 (2)_LI](https://user-images.githubusercontent.com/71714344/111985274-6ab7fb00-8b32-11eb-968c-f84b1a2fa118.jpg)

**Comment by starrohan999** [comment]
please assign it to me.
I am gssoc'21 participant

**Comment by garg3133** [comment]
Yes @starrohan999, we are aware of this issue. A lot of people has pointed this thing out but I was unable to find time or was just lazy to search for the pictures that would go in.

Let me ask someone from the team to find some pictures appropriate for each card and will get back to you.
